opimodel/fonts.py

```diff
@@ -12,11 +12,15 @@
 
 class Font(object):
 
-    def __init__(self, fontface, size, style=REGULAR, pixels=True):
+    def __init__(self, name=None, fontface='Liberation Sans',
```

Comment:
I don't like that we have any named font face here, but it's not that much worse than specify 'Default' as that's a site specific preference anyway. We need to put some text string against  fontface in the xml. 

There shouldn't be any conflict as the font attributes are completely ignored if the named font is found.